script/setup

```diff
@@ -9,31 +9,33 @@ else
 fi
 
 if [ ! -f config/database.yml ]; then
-  echo "Copying database.yml.example over"
+  echo "$0: Copying database.yml.example over"
   cp config/database.yml.example config/database.yml
 fi
 
 # Wipe out logs and scss cache.
-echo "Cleaning out old logs"
+echo "$0: Cleaning out old logs"
 { rm -f log/*
 } >&3 2>&1
 
 # Install dependencies
-echo "Installing libraries and plugins"
+echo "$0: Installing libraries and plugins"
 { gem list -i bundler || gem install bundler
   bundle
   script/install_toxiproxy.sh
 } >&3 2>&1
 
 # Wipe and load the database unless KEEPDB=1 env var is set.
 if [ -z "$KEEPDB" ]; then
-  echo "Reloading the database"
+  echo "$0: Reloading the database"
   { rake db:create:all db:drop:all db:setup --trace
     rake db:test:prepare --trace
   } >&3 2>&1
 fi
 
-echo "Restarting Pow/Passenger"
-touch tmp/restart.txt
+if [ -d tmp ]; then
```

Comment:
I think tmp is used for other things as well? Maybe better to create the directory here if it doesn't exist? I don't want to couple this to pow but seems ok to touch this file.